packages/components/stories/List.js

```diff
@@ -354,6 +354,25 @@ storiesOf('List', module)
 		);
 	})
 
+	.add('Virtualized - table display without click on title', () => {
```

Comment:
Can you move that to special cases below (around empty) and rename it more generically ?
`Virtualized - title without click`